.github/workflows/build-docs.yml

```diff
@@ -12,7 +12,7 @@ defaults:
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: prism-documentation-portal
+  IMAGE_NAME: identus-documentation-portal
```

Comment:
@CryptoKnightIOG, could you involve @milosbackonja into image renaming.
I would recommend him to confirm that this renaming will work as expected.